Changes

```diff
@@ -1,5 +1,32 @@
 Release history for Zonemaster component Zonemaster-GUI 
 
+
+v3.4.0 2021-12-03 (public release version)
+
+ [Features]
+ - Replaces CentOS with Rocky Linux (#280)
+ - Provides translation to error messages from RPC API (#269, #239)
+ - Adds support from information banner (#266)
+ - Adds filter for test history (#253, #251)
+ - Adds run time configuration (#267, #260, #219, #170)
+ - Adds flexible URL (#252, #250)
+ - Updates Danish translation (#248, #244)
```

Comment:
How come Danish translation updates are listed under Features but updates to other languages are listed under Fixes?

Reply:
Mistake. Fixed.